App.tsx

```diff
@@ -1,51 +1,26 @@
 /**
  * Sample React Native App
  * https://github.com/facebook/react-native
- * 
+ *
  * Generated with the TypeScript template
  * https://github.com/emin93/react-native-template-typescript
- * 
+ *
  * @format
  */
 
 import React, {Component} from 'react';
-import {Platform, StyleSheet, Text, View} from 'react-native';
-
-const instructions = Platform.select({
-  ios: 'Press Cmd+R to reload,\n' + 'Cmd+D or shake for dev menu',
-  android:
-    'Double tap R on your keyboard to reload,\n' +
-    'Shake or press menu button for dev menu',
-});
+import {View} from 'react-native';
+import Counter from './Counter';
+import styles from './styles';
 
 interface Props {}
-export default class App extends Component<Props> {
+interface State {}
```

Comment:
Interfaces should start with an I. So IProps, IState. This is a convention in TS to easily visually separate out Classes and Interfaces.